LCM/dsc/engine/ca/CAInfrastructure/WebPullClient.c

```diff
@@ -2810,8 +2827,9 @@ MI_Result MI_CALL Pull_SendStatusReport(_In_ LCMProviderContext *lcmContext,
     MI_Value endTime;
     MI_Uint32 flags;
     int i = 0;
-    const char* commandFormat = NULL;
-    const char* reportText = NULL;
+    const char* commandFormat;
+    const char* reportText;
```

Comment:
just a question: why remove the initialization?

Reply:
I had added it in my last PR. I can put these back in

Reply:
I don't think it matters much, just curious. I try to always initialize autos at the point of declaration. In this case they are initialized later so code will work as is.